renovate/best-practice-all-managers.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "github>konflux-ci/mintmaker//config/renovate/renovate.json",
+    "config:best-practices"
+  ],
+  "enabledManagers": [
+    "tekton",
+    "dockerfile",
+    "rpm",
+    "regex",
+    "argocd",
+    "crossplane",
+    "fleet",
+    "flux",
+    "helm-requirements",
+    "helm-values",
+    "helmfile",
+    "helmsman",
+    "helmv3",
+    "jsonnet-bundler",
+    "kubernetes",
+    "kustomize",
+    "asdf",
+    "fvm",
+    "git-submodules",
+    "hermit",
+    "homebrew",
+    "nix",
+    "osgi",
+    "pre-commit",
+    "vendir",
+    "terraform",
+    "terraform-version",
+    "terragrunt",
+    "terragrunt-version",
+    "tflint-plugin",
+    "pip_requirements",
+    "pip_setup"
```

Comment:
let's also include manager config follow the global config


```json
  "pip_requirements": {
    "additionalBranchPrefix": "{{baseBranch}}/",
    "branchPrefix": "konflux/mintmaker/"
  },
  "pip_setup": {
    "additionalBranchPrefix": "{{baseBranch}}/",
    "branchPrefix": "konflux/mintmaker/"
  },
```

Reply:
Added :+1: